node/node.go

```diff
@@ -199,6 +204,25 @@ func (n *Node) Reconfigure(ctx context.Context, deps resource.Dependencies, conf
 
 	n.gateway = gateway
 
+	// Warn if user's configured capture frequency is more than the expected uplink interval.
+	captureFreq, err := getCaptureFrequencyHzFromConfig(conf)
+	if err != nil {
+		return nil
+	}
+	inter, err := strconv.Atoi(cfg.Interval)
+	if err != nil {
+		return nil
+	}
+	intervalSeconds := (time.Duration(inter) * time.Minute).Seconds()
+	expectedFreq := 1 / intervalSeconds
```

Comment:
is `Interval` required? also need to make sure its nonzero to avoid divide by 0

if its not required then we should only do the freq check if we were given an interval